src/components/lists/ListWrapper.tsx

```diff
@@ -51,15 +49,14 @@ export const ListWrapper = ({
       >
         <Box
           sx={{
+            width: '100%',
             display: 'flex',
             alignItems: { xs: 'flex-start', xsm: 'center' },
             py: '3.6px',
             flexDirection: { xs: 'column', xsm: 'row' },
           }}
         >
-          <Typography component="div" variant={captionSize} sx={{ mr: 4 }}>
-            {title}
-          </Typography>
+          {title}
```

Comment:
Non-functional change, but maybe we update this to `titleComponent` now that we've hiked it up one level like we have subtitleComponent